### DISCUSSION TITLE
Anyway to export Favorite ?

### DISCUSSION THREAD

**Discussion by ghost** [leozide/leocad#883] [Q&A]
Hello there,
I'm working with young children
they are sometimes lost with all these parts.
Is there a way for me to create a new category with a custom parts selection or to share the Favorites ?

Nice job by the way!

**Comment by ghost** [comment]
Is there a prefrences file that I can copy ?

**Reply by leozide** [reply]
You can create categories in the Preferences Dialog and then export/import them:
![image](https://github.com/leozide/leocad/assets/9685066/7a701872-437f-405e-b85a-a29037268545)

Preferences are saved in different places depending on the OS, it's in the registry on Windows or the home folder in Linux.

**Reply by ghost** [reply]
Ok thanks.
Anyway to add parts manually to a category ?
Using the filter is not precise enough for my needs.
For example if I add a category that includes "3005", how can only have the vanilla version and not every variation ("3005pz3", "3005pe2", etc)

**Reply by nathaneltitane** [reply]
> Ok thanks.
> Anyway to add parts manually to a category ?
> Using the filter is not precise enough for my needs.
> For example if I add a category that includes "3005", how can only have the vanilla version and not every variation ("3005pz3", "3005pe2", etc)

@leozide filter was always too fuzzy. he does have a point. any way to implement a more precise set of parameters to narrow down to the exact set?
